common.py

```diff
@@ -1022,6 +1060,10 @@ def apply_cutbasedCRloose(cols):
     cols.cutflow['cutbasedCRloose'] = len(cols)
     return cols
 
+def apply_preselection_minus(cols, minus):
```

Comment:
this goes in https://github.com/boostedsvj/svj_ntuple_processing, not here. this code only analyzes skims, it doesn't create them.